nixos/modules/misc/ids.nix

```diff
@@ -347,6 +347,7 @@ in
       #mailman = 316;  # removed 2019-08-30
       zigbee2mqtt = 317;
       # shadow = 318; # unused
+      etebase-server = 319;
```

Comment:
Should we use dynamic IDs?

Reply:
I'm not really sure how this might affect the server, but I removed the static IDs, and it still seems to work.